### PR TITLE
support multiple file open and save shortcut.

### PR DESCRIPTION
**Pull request by makinux**
Open multiple files and move with arrow key.
And save with Ctrl+s ("original file name".png).

### CODE REVIEW

File: interactive_demo/app.py [hunk 1]
```diff
@@ -161,17 +167,40 @@ def _add_buttons(self):
         FocusHorizontalScale(self.click_radius_frame, from_=0, to=7, resolution=1, command=self._update_click_radius,
                              variable=self.state['click_radius']).pack(padx=10, anchor=tk.CENTER)
 
+    def _setNextImage(self, event):
```
Comment (ksofiyuk) [inline interactive_demo/app.py:170]: Please make the methods naming consistent with [Snake Case](https://en.wikipedia.org/wiki/Snake_case) style.

**Comment by ksofiyuk** [comment]
Thank you for your contribution. Could you add some description of the new functionality in README in Interactive Segmentation Demo section?

File: interactive_demo/app.py (hunk 1 above)
Comment (makinux) [reply interactive_demo/app.py]: Thank you for your advice.
So fix it.

**Comment by makinux** [comment]
@ksofiyuk Added description and video GIFs to README.

**Comment by ksofiyuk** [comment]
I think the addition of 2 Mb GIF is redundant. Just a plain text description would be enough.

**Comment by makinux** [comment]
Thanks for the advice.
Removed the GIF of the operation description.

**Pull request merged**